crates/bevy_render/src/camera/projection.rs

```diff
@@ -131,14 +131,16 @@ impl Default for PerspectiveProjection {
 #[derive(Debug, Clone, Reflect, FromReflect, Serialize, Deserialize)]
 #[reflect(Serialize, Deserialize)]
 pub enum WindowOrigin {
+    /// `(0, 0, 0)` is at the center of the screen.
```

Comment:
```suggestion
    /// `(0, 0)` is at the center of the screen.
```
This affects only 2 dimensions.